wagtail/admin/tests/api/test_pages.py

```diff
@@ -297,42 +297,6 @@ def test_descendant_of_root_doesnt_give_error(self):
 
         self.assertEqual(response.status_code, 200)
 
-    # FOR EXPLORER FILTER
```

Comment:
Rather than removing these tests, shouldn't we be keeping them and pointing them at the new explorer API endpoint instead? (Ditto for `test_explorable_pages_in_explorer` below.)

Reply:
As discussed this morning, it would also be good to add a test to confirm that the explorer API works with a page `type` parameter (which wasn't the case for the old version of the code, because the `explorable_pages` filtering was trying to run a `&` operation between the specific-page queryset and a `Page` queryset).

Reply:
These tests have been added now

Reply:
Thanks - new tests look good to me.